testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json

```diff
@@ -184,5 +184,13 @@
             "sslCertChainPrefix": "x-ssl-client-cert-chain",
             "certificateChainLength": 1
         }
+    },
+
+    "vault": {
+        "provider": "${keycloak.vault.provider:plaintext}",
+        "plaintext": {
+            "dir": "src/test/resources/vault",
```

Comment:
Should this contain full path? Does it work when the testsuite is run e.g. from base dir by `mvn -f testsuite/integration-arquillian/tests/base test`?

Reply:
Ah, I missed that. Ensuring that it works is enough.

Reply:
Just tried it, runs fine.